tests/boot/boot_to_desktop.pm

```diff
@@ -42,4 +42,29 @@ sub test_flags {
     return {fatal => 1, milestone => 1};
 }
 
+sub post_fail_hook {
+    my $self = shift;
+
+    save_screenshot;
```

Comment:
Isn't a screenshot automatically recorded for every post_fail_hook?

Reply:
I can't find a call for that in the opensusebasetest PFH, so if thats the case it's somewhere deeper in OpenQA

Reply:
can you reference a verification run? Then we can check